js/apps/admin-ui/src/components/time-selector/TimeSelector.tsx

```diff
@@ -74,6 +69,16 @@ export const TimeSelector = ({
   const [multiplier, setMultiplier] = useState(defaultMultiplier);
   const [open, setOpen] = useState(false);
 
+  const times = useMemo(() => {
+    const filteredUnits = units.map(
+      (unit) => allTimes.find((time) => time.unit === unit)!
+    );
+    if (filteredUnits.filter((u) => u.multiplier === multiplier).length === 0) {
```

Comment:
Instead of doing a filter and checking the length of the result you can use [`every()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every), which removes the need to check the length and returns a boolean.